client/src/components/Home.svelte

```diff
@@ -0,0 +1,135 @@
+<script>
+  import { onMount } from "svelte";
+
+  let loggedIn = false;
+  let perks = [];
+
+  onMount(async () => {
```

Comment:
will need to add future logic for the other API calls such as login when they are implemented